src/frontend/efiling-frontend/package.json

```diff
@@ -29,7 +29,8 @@
     "react-scripts": "3.4.3",
     "reactstrap": "^8.9.0",
     "regenerator-runtime": "^0.13.7",
-    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/v1.2.1/shared-components-1.2.1.tgz"
+    "shared-components": "https://github.com/bcgov/react-shared-components/releases/download/v1.2.1/shared-components-1.2.1.tgz",
+    "validator": "^13.5.2"
```

Comment:
When adding a new library we should explain it in the PR, what do you think? and add a link to the repo + git stats